options/locale/locale_en-US.ini

```diff
@@ -430,7 +430,7 @@ remove_account_link = Remove linked account
 remove_account_link_desc = Delete this account link will remove all related access for your account. Do you want to continue?
 remove_account_link_success = Account link has been removed successfully!
 
-repos_none = You do not owned any repository
+repos_none = You do not currently have any repositories
```

Comment:
That's a bit wordy and is not consistent with the rest of the strings. I think it should be "You do not own any repositories".